resources/src/js/router/views/signup/index.vue

```diff
@@ -6,68 +6,163 @@
             ></v-text-field>
-            <v-text-field v-model="address1" label="Address 1"></v-text-field>
-            <v-text-field v-model="address2" label="Address 2"></v-text-field>
-            <v-text-field v-model="city" label="City"></v-text-field>
-            <v-text-field v-model="country" label="Country"></v-text-field>
-            <v-text-field v-model="zip" label="Zip Code"></v-text-field>
-            <v-btn color="primary">Submit</v-btn>
-          </v-form>
-        </v-flex>
-        <v-flex pa-1 shrink>
-          <v-btn color="blue-grey" class="white--text">Upload Logo
-            <v-icon right dark>cloud_upload</v-icon>
-          </v-btn>
-        </v-flex>
-      </v-layout>
+            <v-text-field v-model="address_number" prepend-icon="place" label="Address Number"></v-text-field>
+            <v-text-field v-model="street" prepend-icon="map" label="Street"></v-text-field>
+            <v-text-field v-model="city" prepend-icon="location_city" label="City"></v-text-field>
+            <v-text-field v-model="country" prepend-icon="place" label="Country"></v-text-field>
+            <v-text-field v-model="zip_code" prepend-icon="place" label="Zip Code"></v-text-field>
+            <v-text-field v-model="state" prepend-icon="place" label="State"></v-text-field>
+            <img :src="imageUrl" height="150" v-if="imageUrl">
+          </v-flex>
+        </v-layout>
+        <v-spacer></v-spacer>
+        <v-layout>
+          <v-flex>
+            <v-btn color="teal lighten-3" @click="pickFile" v-model="imageFile">Upload Logo</v-btn>
+            <input
+              type="file"
+              style="display: none"
+              ref="image"
+              accept="image/*"
+              @change="onFilePicked"
+            >
+            <v-btn color="primary" @click="submit">Submit</v-btn>
+          </v-flex>
+        </v-layout>
+      </v-form>
     </v-container>
   </v-content>
 </template>
 
 <script>
-import { required } from 'vuelidate/lib/validators';
+import { validationMixin } from "vuelidate";
+import {
+  alpha,
+  alphaNum,
+  email,
+  minLength,
+  numeric,
+  required,
+  sameAs
+} from "vuelidate/lib/validators";
+import axios from "axios";
+import { resolve } from "q";
+
 export default {
-  data: () => ({
-    organization: null,
-    firstName: null,
-    lastname: null,
-    email: null,
-    password: null,
-    validatepassword: null,
-    address1: null,
-    address2: null,
-    city: null,
-    country: null,
-    zip: null
-  }),
+  mixins: [validationMixin],
   validations: {
-      form: {
-          organization: { required },
-          firstName: { required },
-          lastName: { required },
-          email: { required },
-          password: { required },
-          validatePassword: { required },
-          address1: { required },
-          address2: { required },
-          city: { required },
-          country: { required },
-          zip: { required }
+    form: {
```

Comment:
I'm not sure these validations will work... your data properties should also be in a form object with this